tests/pages/login.test.js

```diff
@@ -46,6 +60,8 @@ test("Can log out", async (t) => {
     await t.click(logout);
     const pagePath = await t.eval(() => document.documentURI);
     expect(pagePath).to.equal(url + "/login");
+    const privacyLink = await el("#privacy").exists;
```

Comment:
a little cheeky, hijacking the logout page to see the privacy policy; 
might have been better to have an "i can see the privacy policy link on the logout page" test

Reply:
wouldn't that just duplicate teh whole `Can log out`test and just append two more lines of code? the ones I've added here. Also, when you log out you're redirected to the login page which already contains separate tests about the privacy link. come to think about it, it's redundant to add it even in the "Can log out" test as long as we redirect to login on logout.